Makefile

```diff
@@ -11,6 +11,7 @@ GSOAP_BIN?=$(GSOAP_PREFIX)/bin
 GSOAP_BASE=$(GSOAP_PREFIX)/share/gsoap
 GSOAP_PLUGINS=$(GSOAP_BASE)/plugin
 
+CMAKE_CXX_FLAGS:=$(CXXFLAGS)
```

Comment:
To use it on libv4l2rtspserver cmake, the flags need to be copied before the hard coded values below are added to it.